### DISCUSSION TITLE
Make a quick-install link to use from the terminal

### DISCUSSION THREAD

**Discussion by umaxfun** [ohmyzsh/ohmyzsh#11980] [Ideas]
I use omz everywhere in dev environments, and to install it to another environment I do the following:
1. Get the installation script's URL from the terminal
```
➜  ~ curl -L ohmyz.sh | grep curl
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0          the command-line with either curl or wget.</p>
          <li id="curl-tab" class="curl-tab active-tab">Install oh-my-zsh via curl</li>
          <div class="curl">
            <p class="code">sh -c "$(curl -fsSL https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"</p>
100 37188  100 37188    0     0  83714      0 --:--:-- --:--:-- --:--:-- 85686
```
2. Using the mouse copy/paste the curl command into the terminal =/

We could use a quick URL for this case to be like this `$(curl -L ohmyz.sh/i)` and voila, it's installed.

We just need to put the install command to the separate file on the website named `i`

**Comment by carlosala** [comment]
You can use `install.ohmyz.sh` alternatively.

**Reply by umaxfun** [reply]
Wow, it is definitely worth writing it on the website clearly! I'm struggling with my method literally for years :)

Thank you for your work!

**Reply by carlosala** [reply]
It's more recommended to use the one of GitHub still. The other is just a mirror. Thanks!
